components/layout/index.js

```diff
@@ -1,19 +1,64 @@
-import React from 'react';
-import * as layout from './index.module.css';
-
-const Layout = ({children}) => {
-        return(
-            <div className={layout.layout}> 
-            <div className={layout.header}>
-                New Abhishek
-            </div>
-            <div className={layout.body}>
-                {children}
-            </div>
-            <div className={layout.footer}>
-            ©2020 by New Abhishek. Proudly Created by meShakti
-            </div>
-            </div>
-        )
+import React, { useState } from "react";
+import * as layout from "./index.module.css";
+import PropTypes from "prop-types";
+
+
+
+
+const Layout = ({ children }) => {
+	
+	const [isExpanded, toggleExpansion] = useState(false);
+  
+  
+	return (
+
+		<div className={layout.layout}>
+
+			<nav className="flex items-center justify-between flex-wrap bg-black p-6">
```

Comment:
Make a header component and move this logic there

Reply:
Done